fastly/block_fastly_service_backend.go

```diff
@@ -37,6 +37,12 @@ func (h *BackendServiceAttributeHandler) GetSchema() *schema.Schema {
 			Required:    true,
 			Description: "An IPv4, hostname, or IPv6 address for the Backend",
 		},
+		"auto_loadbalance": {
```

Comment:
You might need to rebase `main` branch as this was updated recently...
https://github.com/fastly/terraform-provider-fastly/pull/657